calyx-ir/src/control.rs

```diff
@@ -67,6 +105,34 @@ impl GetAttributes for If {
 
 /// Data for the `if` control statement.
 #[derive(Debug)]
+pub struct StaticIf {
+    /// Port that connects the conditional check.
+    pub port: RRC<Port>,
+
+    /// latency field
+    pub latency: u64,
```

Comment:
Worth describing what it means for an `if` to have a latency if the two branches of `if` might not be balanced (i.e., take different amounts of time)